tensorflow/lite/core/api/flatbuffer_conversions.cc

```diff
@@ -1514,6 +1508,20 @@ TfLiteStatus ParseRsqrt(const Operator*, ErrorReporter*, BuiltinDataAllocator*,
   return kTfLiteOk;
 }
 
+// We have this parse function instead of directly returning kTfLiteOk from the
+// switch-case in ParseOpData because this function is used as part of the
+// selective registration for the OpResolver implementation in micro.
```

Comment:
remove this comment since it is only relevant for empty functions (like ParseSin).

Reply:
Done